Modules/_sqlite/microprotocols.c

```diff
@@ -58,8 +58,8 @@ pysqlite_microprotocols_add(PyTypeObject *type, PyObject *proto, PyObject *cast)
     PyObject* key;
     int rc;
 
-    if (proto == NULL) proto = (PyObject*)pysqlite_PrepareProtocolType;
-
+    assert(type != NULL);
+    assert(proto != NULL);
```

Comment:
There is only one user of `pysqlite_microprotocols_add`; `proto` will never be `NULL`.
https://github.com/python/cpython/blob/2780df478129c4f1042df2868b6adf2bf507cd6e/Modules/_sqlite/module.c#L179

Converting the sanity check to an assert instead.